test/integration/template_database_test.go

```diff
@@ -3221,3 +3221,52 @@ func TestDatabaseStatefulSetVolumeSnapshot(t *testing.T) {
 		}
 	})
 }
+func TestDatabaseAdminAffinityLabels(t *testing.T) {
+	// Path to the helm chart we will test
+	helmChartPath := testlib.DATABASE_HELM_CHART_PATH
+
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"admin.affinityLabels":           "zone host",
+			"database.sm.noHotCopy.replicas": "1",
+			"database.sm.hotCopy.replicas":   "1",
+		},
+	}
+
+	// Run RenderTemplate to render the template and capture the output.
+	output := helm.RenderTemplate(t, options, helmChartPath, "release-name", []string{"templates/statefulset.yaml"})
+
+	for _, obj := range testlib.SplitAndRenderStatefulSet(t, output, 2) {
+		require.NotEmpty(t, obj.Spec.Template.Spec.Containers)
+
+		container := obj.Spec.Template.Spec.Containers[0]
+
+		found := false
+		for i, arg := range container.Args {
+			if arg == "--admin-affinity-label-keys" {
+				assert.Equal(t, "zone host", container.Args[i+1])
+				found = true
+			}
+		}
+
+		assert.True(t, found)
```

Comment:
Nit: Can be replaced with:
```
assert.True(t, testlib.ArgContains(args, "--admin-affinity-label-keys"))
assert.True(t, testlib.ArgContains(args, "zone host"))
```

Reply:
I want to validate that the args are in the right order and don't have any other flags in between (for example, I did not mess up the order and put `zone host --admin-affinity-label-keys` or I inserted another arg later and ended up with `--admin-affinity-label-keys --some-other-flag zone host`).